Source/WebCore/dom/IgnoreOpensDuringUnloadCountIncrementer.h

```diff
@@ -33,7 +33,7 @@ class IgnoreOpensDuringUnloadCountIncrementer {
     WTF_MAKE_NONCOPYABLE(IgnoreOpensDuringUnloadCountIncrementer);
 public:
     explicit IgnoreOpensDuringUnloadCountIncrementer(Document* document)
```

Comment:
This should probably be at least renamed.  I actually think we should replace its unsigned* with a ScopeExit that captures a WeakPtr&lt;Document&gt;

Reply:
I  renamed the helper class. I am not really seeing how to apply the ScopeExit here, but anyway I think it would remove the nice ctor/dtor symmetry. I do prefer a bit to hold the Document rather than an unsigned*, so I made that change.